plugins/woocommerce-blocks/assets/js/blocks/collection-filters/inner-blocks/attribute-filter/components/attribute-dropdown.tsx

```diff
@@ -1,36 +1,28 @@
 /**
  * External dependencies
  */
-import styled from '@emotion/styled';
-import FormTokenField from '@woocommerce/base-components/form-token-field';
 import { __, sprintf } from '@wordpress/i18n';
 import { Icon, chevronDown } from '@wordpress/icons';
 
+/**
+ * Internal dependencies
+ */
+import { PreviewDropdown } from '../../components/preview-dropdown';
+
 type Props = {
 	label: string;
 	textColor: string;
 };
 
-export const AttributeDropdown = ( { label, textColor }: Props ) => {
-	const StyledFormTokenField = textColor
-		? styled( FormTokenField )`
-				.components-form-token-field__input::placeholder {
-					color: ${ textColor } !important;
-				}
-		  `
-		: FormTokenField;
-
+export const AttributeDropdown = ( { label }: Props ) => {
 	return (
 		<div className="wc-block-attribute-filter style-dropdown">
-			<StyledFormTokenField
-				suggestions={ [] }
+			<PreviewDropdown
 				placeholder={ sprintf(
 					/* translators: %s attribute name. */
 					__( 'Select %s', 'woocommerce' ),
 					label
 				) }
```

Comment:
The Dropdown component currently always shows `Select an option` on the frontend, it'd be nice to add this ability to customize the placeholder (in another PR) on the frontend.

Reply:
Yep that should be a quick fix! I'll add a task for that.

Reply:
https://github.com/woocommerce/woocommerce/issues/43205